content/en/dashboards/widgets/split_graph.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Split Graph Widget
+kind: documentation
+widget_type: split_graph
```

Comment:
```suggestion
widget_type: split_group
```
Kinda confusing, but it looks like you have to use `split_group` for the API. This is so the tables at the bottom of the page load correctly.

Reply:
Thanks for catching this Heston!